src/DMLQueryBuilder.php

```diff
@@ -61,6 +62,19 @@ public function resetSequence(string $table, int|string|null $value = null): str
 EXECUTE autoincrement_stmt";
     }
 
+    public function update(
+        string $table,
+        array $columns,
+        array|string|ExpressionInterface $condition,
+        array|string|ExpressionInterface|null $from = null,
+        array &$params = []
+    ): string {
+        /**
+         * MySQL does not support UPDATE ... FROM ...
+         */
```

Comment:
It supports using another syntax
`UPDATE table_name, table_from SET table_name.field=table_from.field`